addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -41,7 +41,14 @@ def _holiday_status_id_domain(self):
             return [('valid', '=', True), ('allocation_type', '!=', 'no')]
         return [('valid', '=', True), ('allocation_type', '=', 'fixed_allocation')]
 
-    name = fields.Char('Description', compute='_compute_description', inverse='_inverse_description', search='_search_description', compute_sudo=False)
+    name = fields.Char(
+        'Description',
+        store=True,
```

Comment:
Careful with that, model changes are not allowed in stable releases. See the [stable policy](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean) ;)
Storing the fields requires a new column in the database.
This should be done in the master branch.

Reply:
Right!
Other solution found after discussion : remove this field from the 'group by' list as it useless to group by description.